homeassistant/components/eight_sleep/config_flow.py

```diff
@@ -53,13 +54,18 @@ async def async_step_user(
 
         await self.async_set_unique_id(f"{DOMAIN}.{user_input[CONF_USERNAME]}")
         self._abort_if_unique_id_configured()
-        await eight.fetch_token()
-        if not eight.token:
+        try:
+            await eight.fetch_token()
+        except RequestError as err:
             return self.async_show_form(
```

Comment:
Usually we abort and log if we can't import since otherwise they end up with an unexpected flow which confuses users